src/scene/skybox/sky.js

```diff
@@ -109,11 +115,37 @@ class Sky {
         return this._center;
     }
 
+    /**
+     * Whether depth writing is enabled for the sky. Defaults to false.
+     *
+     * Writing a depth value for the skydome is supported when its type is not
+     * {@link SKYTYPE_INFINITE}. When enabled, the depth is written during a prepass render pass and
+     * can be utilized by subsequent passes to apply depth-based effects, such as Depth of Field.
+     *
+     * Note: For the skydome to be rendered during the prepass, the Sky Layer must be ordered before
+     * the Depth layer, which is the final layer used in the prepass.
+     *
+     * @type {boolean}
+     */
+    set depthWrite(value) {
+        if (this._depthWrite !== value) {
+            this._depthWrite = value;
+            if (this.skyMesh) {
+                this.skyMesh.depthWrite = value;
+            }
+        }
+    }
+
+    get depthWrite() {
```

Comment:
Don't forget to document the getter as well! 😄

Reply:
done thanks